model/countries/AU/AU-parsing-rules.yaml

```diff
@@ -0,0 +1,304 @@
+  type: in-building-location
+  input: "suite 75 ug"
+  output:
+    in-building-location: "suite 75 ug"
+    unit: "suite 75"
+    unit-type: "suite"
+    unit-name: "75"
+- id: "Test 4"
+  type: building-location
+  input: "75 Lane Street"
+  output:
+    street: "Lane Street"
+    building: "75"
+- id: "Test 5"
+  type: building-location
+  input: "189 Great Eastern Highway"
+  output:
+    street: "Great Eastern Highway"
+    building: "189"
+- id: "Test 6"
+  type: building-location
+  input: "189 Great Eastern Highway"
+  output:
+    street: "Great Eastern Highway"
+    building: "189"
+- id: "Test 7"
+  type: street-address-alternative-1
+  input: "Unit 7 Level 8  189 Great Eastern Highway"
+  output:
+    street-address-alternative-1: "Unit 7 Level 8  189 Great Eastern Highway"
+    in-building-location: "Unit 7 Level 8"
+    building-location: "189 Great Eastern Highway"
+    unit: "Unit 7"
+    unit-type: "Unit"
+    unit-name: "7"
+    floor: "8"
+    street: "Great Eastern Highway"
+    building: "189"
+- id: "Test 8"
+  type: street-address-alternative-1
+  input: "suite 5 fl 10  189 Great Eastern Highway"
+  output:
+    street-address-alternative-1: "suite 5 fl 10  189 Great Eastern Highway"
+    in-building-location: "suite 5 fl 10"
+    building-location: "189 Great Eastern Highway"
+    unit: "suite 5"
+    unit-type: "suite"
+    unit-name: "5"
+    floor: "10"
+    street: "Great Eastern Highway"
+    building: "189"
+- id: "Test 9"
+  type: street-address-alternative-1
+  input: "Unit 7  189 Great Eastern Highway"
+  output:
+    street-address-alternative-1: "Unit 7  189 Great Eastern Highway"
+    building-location: "189 Great Eastern Highway"
+    street: "Great Eastern Highway"
+    building: "189"
+    in-building-location: "Unit 7"
+    unit: "Unit 7"
+    unit-type: "Unit"
+    unit-name: "7"
+- id: "Test 10"
+  type: street-address-alternative-1
+  input: "Floor 10  189 Great Eastern Highway"
+  output:
+    street-address-alternative-1: "Floor 10  189 Great Eastern Highway"
+    building-location: "189 Great Eastern Highway"
+    street: "Great Eastern Highway"
+    building: "189"
+    in-building-location: "Floor 10"
+    floor: "10"
+- id: "Test 11"
+  type: street-address-alternative-1
+  input: "189 Great Eastern Highway"
+  output:
+    street-address-alternative-1: "189 Great Eastern Highway"
+    building-location: "189 Great Eastern Highway"
+    street: "Great Eastern Highway"
+    building: "189"
+- id: "Test 12"
+  type: in-building-location
+  input: "Unit 7 Floor 15"
+  output:
+    in-building-location: "Unit 7 Floor 15"
+    unit: "Unit 7"
+    unit-type: "Unit"
+    unit-name: "7"
+    floor: "15"
+- id: "Test 13"
+  type: in-building-location
+  input: "Unit 7"
+  output:
+    in-building-location: "Unit 7"
+    unit: "Unit 7"
+    unit-type: "Unit"
+    unit-name: "7"
+- id: "Test 14"
+  type: in-building-location
+  input: "Floor 7"
+  output:
+    in-building-location: "Floor 7"
+    floor: "7"
```

Comment:
I think that I have also seen a spelling "7 / 189 Great Eastern Highway". Is that right? Do you plan to support that as well?

Reply:
Added some more tests and adjusted the changes! PTAL